test/project_test.dart

```diff
@@ -9,57 +9,46 @@ library linter.test.project;
 import 'dart:io';
 
 import 'package:linter/src/project.dart';
-import 'package:unittest/unittest.dart';
+import 'package:test/test.dart';
 
-main() {
-  groupSep = ' | ';
-
-  defineTests();
-}
-
-defineTests() {
```

Comment:
For why?

In a world where you want to set up config and aggregate things, it makes sense. In our new world, config should be completely separate, so it seems a tad gratuitous